ArcGISRuntimeSDKQt_CppSamples/Analysis/DistanceMeasurementAnalysis/DistanceMeasurementAnalysis.qml

```diff
@@ -93,8 +93,22 @@ DistanceMeasurementAnalysisSample {
                 color: "white"
             }
             ComboBox {
+                id: comboBox
+                property int modelWidth: 0
```

Comment:
would it be a good idea to have the initial `modelWidth` set to a non-zero value?

In the (admittedly impossible/unlikely) case that your model starts empty it will at least be a more usable control...